src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -111,7 +111,7 @@ public async Task ExecuteAsync(ExportJobRecord exportJobRecord, WeakETag weakETa
                 // from the search result.
                 var queryParametersList = new List<Tuple<string, string>>()
                 {
```

Comment:
nit: Since we are now using `_exportJobRecord` for almost everything in `ExportJobTask`, I am wondering whether it might be better to update the `ConnectAsync` and other downstream methods to take in a single string instead of an `ExportJobConfiguration`. That way we use the `ExportJobConfiguration` only for the initial comparison and then base everything else off of `ExportJobRecord`

Reply:
I think that would be a good idea, but it would not be a small refactor. I'll make anther work item for it.